lib/decorators/party_name_attributes.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+require 'scraped'
+
+class PartyNameAttributes < Scraped::Response::Decorator
```

Comment:
If the name of a Decorator doesn't make it very clear what it's doing, then that should at least be documented in a comment.